packages/internet-header/src/components/post-search/post-search.tsx

```diff
@@ -377,6 +396,17 @@ export class PostSearch implements HasDropdown, IsFocusable {
                         onKeyDown={e => this.handleKeyDown(e)}
                       />
                       <label htmlFor="searchBox">{translations.flyoutSearchBoxFloatingLabel}</label>
+                      <button
+                        onClick={() => this.handleClearSearchbar()}
+                        class="visually-hidden clear-search-button"
+                        type="reset"
+                        aria-label="Clear input"
```

Comment:
This label could be more explicit, for example “Clear search term”.
It also needs to be translated so that it is accessible to all users, check bellow `searchSubmit` label for an example.